fixture/vcr_cassettes/twilio_sms_failure.json

```diff
@@ -1,7 +1,7 @@
 [
   {
     "request": {
-      "body": "Body=Hello+there&From=%2B15551234567&To=%2B15557654321",
+      "body": "Body=Hello+there&From=%2B15551234567&MessageId=&Provider=&To=%2B15557654321",
```

Comment:
I don't believe these two params (`MessageId` & `Provider`) are valid params to send to Twilio. I think we should drop them when constructing the params here https://github.com/seated/apus/blob/main/lib/apus/adapters/twilio_adapter.ex#L50?